tests/table.spec.ts

```diff
@@ -139,6 +139,176 @@ test.describe('tests complex table operations', () => {
 
     // Check snapshot
     await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+
+    await page
+      .locator('.iris-grid')
+      .locator('data-testid=btn-page-close')
+      .first()
+      .click();
+
+    await expect(page.locator('.iris-grid .table-sidebar')).toHaveCount(0);
+  });
+
+  test('can conditional format', async ({ page }) => {
+    await page.goto('');
+    const consoleInput = page.locator('.console-input');
+    await consoleInput.click();
+
+    const command = `${makeTableCommand(
+      undefined,
+      TableTypes.StringAndNumber
+    )}`;
+
+    await pasteInMonaco(consoleInput, command);
+    await page.keyboard.press('Enter');
+
+    // Wait for the panel to show
+    await expect(page.locator('.iris-grid-panel')).toHaveCount(1);
+
+    // Wait until it's done loading
+    await expect(page.locator('.iris-grid-panel .loading-spinner')).toHaveCount(
+      0
+    );
+
+    // Model is loaded, need to make sure table data is also loaded
+    await expect(
+      page.locator('.iris-grid .iris-grid-loading-status')
+    ).toHaveCount(0);
+
+    const tableOperationsMenu = page.locator(
+      'data-testid=btn-iris-grid-settings-button-table'
+    );
+    await tableOperationsMenu.click();
+
+    // Wait for Table Options menu to show
+    await expect(page.locator('.table-sidebar')).toHaveCount(1);
```

Comment:
This is boilerplate code used by all your sidebar tests, you can just put this into a [beforeEach](https://playwright.dev/docs/api/class-test#test-before-each) within a [describe](https://playwright.dev/docs/api/class-test#test-describe-1) block for the sidebar tests, e.g.:
```
test.describe('table sidebar', () => {
  test.beforeEach(async ({page}) => {
    // ... open the table and sidebar
  });

  test('conditional formatting', async () => { ... });
  test('organize columns', async () => { ... });
  ...
});
```